### PR TITLE
Add goal meter to roadmap page

### PR DESCRIPTION
**Pull request by broskoski**
Screenshot 2019-05-06 17 44 44

### CODE REVIEW

File: src/v2/components/UI/Meter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  )
+}
+
+Meter.defaultProps = {
+  borderRadius: '0.5em',
+  bg: 'gray.light',
+  borderColor: 'gray.regular',
+  p: 4,
+  mb: 0,
+}
```
Comment (dzucconi) [inline src/v2/components/UI/Meter/index.tsx]: You could just inline these in the argument destructuring `borderRadius = '0.5em'` etc...

File: src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  goalDate: string
+}
+
+export const GoalMeter: React.FC<GoalMeterProps> = ({
+  currentMrr,
+  goalMrr,
+  goalDate,
+}) => {
+  return (
+    <React.Fragment>
```
Comment (dzucconi) [inline src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx]: Been preferring the `<></>` syntax for fragments.

Comment (broskoski) [reply src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx]: ++

File: src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      <Meter
+        bg="gray.light"
+        borderColor="transparent"
+        borderRadius="1.5em"
+        startColor="state.premium"
+        endColor="state.supporter"
```
Comment (dzucconi) [inline src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx]: 💯

File: src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          </Text>
+          <Text f={1}>
+            <strong>${currentMrr}</strong> Monthly recurring revenue
+          </Text>
+        </Box>
+        <Box>
+          <Text f={1} align="right">
+            <strong>Goal by {new Date(goalDate).toDateString()}</strong>
+          </Text>
+          <Text f={1} align="right">
```
Comment (dzucconi) [inline src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx]: What's this align prop? Does styled-system accept that as an alias for `textAlign`?

Comment (dzucconi) [reply src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx]: `fontWeight="bold"` instead of `<strong>`, one less tag

Comment (broskoski) [reply src/v2/pages/about/RoadmapPage/components/GoalMeter/index.tsx]: ++

File: src/v2/components/UI/Meter/index.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+interface MeterProps {
+  startColor: string
+  endColor: string
+  amount: number
+  limit: number
+  borderRadius?: string
+  bg?: string
+  borderColor?: string
+  p?: number
+  mb?: number
```
Comment (damassi) [inline src/v2/components/UI/Meter/index.tsx]: All common styled-system props can be imported via `@types/styled-system`: 

`import { SpaceProps, etc } from 'styled-system'`

```js
interface MeterProps extends SpaceProps {
  ...
}
```

Comment (damassi) [reply src/v2/components/UI/Meter/index.tsx]: This maps 1:1 with available types: https://github.com/styled-system/styled-system/blob/master/docs/table.md

Comment (broskoski) [reply src/v2/components/UI/Meter/index.tsx]: ++ thanks @damassi

**Pull request merged**